ocfweb/docs/docs/staff/policy.md

```diff
@@ -0,0 +1,83 @@
+[[!meta title="Staff policy"]]
+
+
+Staff members shall be chosen by the OCF Decision Making Process. [per OCF
+Board decision on 4/13/89]
+
+
+## Background
+
+The OCF Decision Making Process spells out the powers of BoD, the GMs, and the
+SMs; however, it doesn’t explain what powers other staffers have. The main
+purpose of this policy is to delineate what things staff can do on their own,
+and what things staff require special permission from the SM to do.
+
+## Authorization
+
+Staff is authorized to act independently of the SMs unless otherwise specified
+here or elsewhere. This is so decisions do not have to be delayed by the
+bureaucracy of having to ask the SMs for approval for all decisions.
```

Comment:
> We have this language in our current staff policy.

Do we actually have a current staff policy? Please share, if so.

I should have said "obvious" instead of "superfluous" though. As in, I think anyone reading can infer the purpose of granting autonomy to staffers. But if we already have that line somewhere else, I'm not going to complain.